gatsby-node.js

```diff
@@ -134,6 +134,10 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
     "src/templates/workshop-single.js"
   );
 
+  // const MultiWorkshopTemplate = path.resolve(
+  //   "src/sections/Careers/Careers-Programs-grid/index.js"
+  // );
+
```

Comment:
Was this commented out or just used for testing?

Reply:
Hi @Nikhil-Ladha,

Yes, that commented part was used for testing to resolve another quirk about MDX v2 and gatsby-plugin-mdx v4, which was the removal of the [`MDXrenderer` function](https://github.com/gatsbyjs/gatsby/discussions/34714) which does not seem to have a clear workaround.

I will add descriptive comments to these kind of items for reviewers in the future.

Thanks,
Randy